Dockerfile.lint

```diff
@@ -1,5 +1,5 @@
 ARG ALPINE_VERSION=3.10
```

Comment:
Maybe could we upgrade to alpine 3.10.1 here too?

Reply:
Unfortunately no... from https://hub.docker.com/_/golang :
> 1.12.9-alpine3.10, 1.12-alpine3.10, 1-alpine3.10, alpine3.10, 1.12.9-alpine, 1.12-alpine, 1-alpine, alpine

`alpine3.10` is the only available...